### ISSUE TITLE
Why is this app not in the playstore or in the f-droid store ?

### ISSUE THREAD

**Issue by DanielGermann** [unsurv/unsurv-android#2]
Yeah i was wondering why this app is only available on github. Are you plannig to releases it also in appstores ?

**Comment by unsurv** [comment]
Some version of the app (probably a rebuilt version to decrease complexity) will be available within the next year.

**Comment by muelli** [comment]
yeah, it would be great and greatly increase the number of people who could casually map some cameras.

**Comment by unsurv** [comment]
Referring to #2 what do you think about a browser based tool? Cross-platform + easy accessibility / barrier of entry.
You can expect some kind of prototype between Christmas and New Years.
